src/components/SelectedMoviesCard/SelectedMoviesCard.js

```diff
@@ -0,0 +1,25 @@
+import './SelectedMoviesCard.css'
+
+
+const SelectedMoviesCard = ( {selectedMovie}) => {
+
+  return (
+    <div className='sel-movie-card'>
+      <section>
+        <img  className='sel-movie-backdrop-img' src={selectedMovie.backdrop_path}></img>
+      </section>
+      <section className='sel-movie-detail-section'>
+        <img className='sel-movie-poster-img' src={selectedMovie.poster_path}></img>
+        <h2>{selectedMovie.title}</h2>
+        <div>
+          <p>Rating: {selectedMovie.average_rating}</p>
```

Comment:
I know some of the individual movie data is inconsistent across the movies.  How will you handle missing data for any of the individual movie data points?